README.rst

```diff
@@ -1,86 +1,159 @@
+.. _header-n957:
+
 SageMaker Containers
 ====================
 
-SageMaker Containers contains common functionality necessary to create a
-container compatible with SageMaker. It can be simply used by any
-container by just installing the module:
+SageMaker Containers gives you tools to create SageMaker-compatible Docker containers, and has additional tools for letting you create Frameworks
+(SageMaker-compatible Docker containers that can run arbitrary Python or shell scripts).
 
-.. code:: bash
+Currently, this library is used by the following containers: `TensorFlow
+Script Mode <https://github.com/aws/sagemaker-tensorflow-container/tree/script-mode>`__,
+`MXNet <https://github.com/aws/sagemaker-mxnet-container>`__,
+`PyTorch <https://github.com/aws/sagemaker-pytorch-container>`__,
+`Chainer <https://github.com/aws/sagemaker-chainer-container>`__, and
+`Scikit-learn <https://github.com/aws/sagemaker-scikit-learn-container>`__.
 
-   pip install sagemaker-containers
+.. contents::
 
-SageMaker Containers gives you tools to create SageMaker-compatible
-containers, and has additional tools for letting you create Frameworks
-(SageMaker-compatible containers that can run arbitrary Python scripts).
+.. _header-n1174:
 
-Currently, this library is used by:
+Getting Started
+===============
 
-- `SageMaker TensorFlow Script Mode Container <https://github.com/aws/sagemaker-tensorflow-container/tree/script-mode>`__
-- `SageMaker MXNet Container <https://github.com/aws/sagemaker-mxnet-container>`__
-- `SageMaker PyTorch Container <https://github.com/aws/sagemaker-pytorch-container>`__
-- `SageMaker Chainer Container <https://github.com/aws/sagemaker-chainer-container>`__
+.. _header-n962:
 
-Getting Started - Executing User Scripts on Amazon SageMaker
-------------------------------------------------------------
+Creating a container using SageMaker Containers
+-----------------------------------------------
 
-The objective of this tutorial is to explain how a script is executed
-inside any SageMaker-compatible container using **SageMaker
-Containers**.
+Here we'll demonstrate how to create a Docker image using SageMaker Containers in order to show the simplicity of using this library.
 
-Creating the training job
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Let's suppose we need to train a model with the following training script train.py using TF 2.0 in SageMaker:
```

Comment:
I'd put "train.py" either in quotes or backticks

Reply:
I added double backticks in all file names.